pkg/ui/BUILD.bazel

```diff
@@ -13,3 +13,17 @@ go_library(
         "@com_github_elazarl_go_bindata_assetfs//:go-bindata-assetfs",
     ],
 )
+
+filegroup(
```

Comment:
get all `node_module`'s packages except local linked packages

Reply:
`node_modules` is a `gitignore`d directory, so this seems wrong. What are you trying to do here?

Reply:
Bazel treats `node_modules` as an external repository and used to include external dependencies:
```
workspace(
    name = "cockroach",
    managed_directories = {
       "@npm": ["pkg/ui/node_modules"]
    },
)
```
except external packages, it also included symlinked local packages
```
|- workspaces
    |- cluster-ui
    |- db-console
|- node_modules
    |- ... 
    |- cluster-ui (symlink) 
```
and to avoid circular dependencies for cluster-ui rules, we need all packages from node_modules except `cluster-ui`.